docs/guides/validation/validation_actions/how_to_collect_openlineage_metadata_using_a_validation_action.md

```diff
@@ -29,47 +29,40 @@ Steps
     % pip3 install openlineage-common
     ```
 
-2. Update the action_list key in your Validation Operator config.
+2. Update the action_list key in your Checkpoint config.
 
-    Add the ``OpenLineageValidationAction`` action to the ``action_list`` key of the ``ActionListValidationOperator`` config in your ``great_expectations.yml``.
+    Add the ``OpenLineageValidationAction`` action to the end of your Checkpoint's ``action_list``.
```

Comment:
```suggestion
    Add the ``OpenLineageValidationAction`` Action to the end of your Checkpoint's ``action_list``.
```
Actions such as these fall under the guidance for technical terms, so we want to title case them. (Originally, the technical term was Validation Action, but in the universal map update that is going to be shortened to Action.)